app/models/user.rb

```diff
@@ -5,9 +5,21 @@ class User < ApplicationRecord
          :recoverable, :rememberable, :validatable,
          :omniauthable, omniauth_providers: [:facebook]
         
-  has_attached_file :avatar, styles: { medium: '300x300>', thumb: '100x100>' }
+  has_one_attached :avatar 
   has_many :todo_lists
-  validates_attachment_content_type :avatar, content_type: /\Aimage\/.*\Z/
+  validate :avatar_validation
+
+  def avatar_validation
```

Comment:
Well done, we need to wait until ActiveStorage gets validations out of the box.

Or... try to create a contribution for that 🤓 😅